src/event-log.c

```diff
@@ -0,0 +1,398 @@
+#include <errno.h>
+#include <glib.h>
+#include <gio/gio.h>
+#include <glib/gstdio.h>
+#if ENABLE_JSON
+#include <json-glib/json-glib.h>
+#endif
+
+#include "context.h"
+#include "event-log.h"
+
+static const gchar *supported_event_types[] = {
+	"all",
+	R_EVENT_LOG_TYPE_BOOT,
+	R_EVENT_LOG_TYPE_INSTALL,
+	R_EVENT_LOG_TYPE_SERVICE,
+	R_EVENT_LOG_TYPE_WRITE_SLOT,
+	R_EVENT_LOG_TYPE_BOOT_SELECTION,
```

Comment:
Perhaps some comments to clarify the respective expected content?

Reply:
These are `#define`d in `include/event-log.h`, so the comments should go there. Still applies to the current version.

Reply:
Without actually using it, the scope is probably still a bit vague but tried to be a bit clearer.

Might need some adaptions and extensions when adding actual log message printouts later anyway.